### DISCUSSION TITLE
I'm new to github

### DISCUSSION THREAD

**Discussion by TheA-Code** [community/community#42449] [New to GitHub] [answered]
What is the difference between Git and GitHub?

**Comment by Luis-Angel-Santos** [comment, marked as answer]
First of all, welcome to GitHub! 🎉

To summarize the difference between git vs GitHub:

- Git is a local VCS software that allows developers to save snapshots of their projects over time. It is generally best for individual use.

- GitHub is a web-based platform that incorporates the version control features of git so they can be used collaboratively. It also includes project and team management features, as well as opportunities for networking and social coding.

**Reply by TheA-Code** [reply]
Thanks a lot

**Reply by micman69** [reply]
Cheers for the info

**Reply by gholamrazavafaei824** [reply]
کیت توضیح کاری است که هاب انجام می‌دهد یا کیت عامل کار است وهاب انجام دهنده کار

**Comment by Zebbra777** [comment]
Halo

**Reply by Raul7671** [reply]
HALO

**Comment by cporter-TIME** [comment]
How can I speak to a representative about pricing or get a quote for my organization?

**Reply by nyakase** [reply]
https://github.com/enterprise/contact

**Reply by gholamrazavafaei824** [reply]
با ایمیل یاپیام یاتلفن

**Comment by gholamrazavafaei824** [comment]
[]() مدیریت حسابداری
